apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -633,7 +669,7 @@ const BookingPage = ({
                       {t("location")}
                     </span>
                     {locations.map((location, i) => {
-                      const locationString = locationKeyToString(location);
+                      const locationString = locationKeyToString(location as unknown as Props);
```

Comment:
We avoid this unless is strictly necessary. Let's figure out why the type are being lost. We cannot merge in this state.